rstudio/init.sh

```diff
@@ -0,0 +1,30 @@
+# download rstudio 
+wget http://download2.rstudio.org/rstudio-server-rhel-0.99.446-x86_64.rpm
+sudo yum install --nogpgcheck -y rstudio-server-rhel-0.99.446-x86_64.rpm
+
+# restart rstudio 
+rstudio-server restart 
+
+# add user for rstudio 
+adduser rstudio
```

Comment:
So if I'm reading this right, there is no password set for this user by default ? Do we expect users to run `passwd rstudio` ? (FWIW that's fine by me as long as we can document that and show that message to the users)